pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -486,15 +486,13 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 	// Before starting to process DomainConfig, domainmgr should (in this order):
 	//   1. wait for NIM to publish DNS to learn which ports are used for management
 	//   2. wait for PhysicalIOAdapters (from zedagent) to be processed
-	//   3. wait for NIM to finalize testing of selected DPC
-	//   4. wait for capabilities information from hypervisor
-	// Note: 2. and 3. can also execute in the reverse order.
+	//   3. wait for capabilities information from hypervisor
 	// Note: 4 may come in any order
```

Comment:
Looks like you removed 4 from the comment. Was that accidental?

Reply:
Mistake on my side - 4th point is now 3rd and line "4 may come in any order" should have been updated. Fixed.